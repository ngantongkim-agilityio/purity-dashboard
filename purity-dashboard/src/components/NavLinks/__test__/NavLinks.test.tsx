import { render } from '@testing-library/react';

// Components
import { NavLinks } from '..';

describe('NavLinks component', () => {
  it('Should render NavLinks component with default props', () => {
    const { container } = render(<NavLinks links={[]} />);

    expect(container).toMatchSnapshot();
  });
});
