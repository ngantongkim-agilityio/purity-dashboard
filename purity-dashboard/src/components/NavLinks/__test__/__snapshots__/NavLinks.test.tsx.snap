// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLinks component Should render NavLinks component with default props 1`] = `
<div>
  <nav
    class="flex grow flex-row md:grow-0 justify-between md:flex-col space-x-4 md:space-x-0 md:space-y-2 sm:mr-16 md:mr-0"
  />
</div>
`;
